source/FreeRTOS_IPv6.c

```diff
@@ -64,15 +130,34 @@ const struct xIPv6_Address FreeRTOS_in6addr_loopback = { { 0, 0, 0, 0, 0, 0, 0,
  */
 BaseType_t xIsIPv6Multicast( const IPv6_Address_t * pxIPAddress )
 {
-    BaseType_t xReturn;
+    BaseType_t xReturn = pdFALSE;
```

Comment:
Instead of xIsIPv6Multicast, xIsIPv6AllowedMulticast seems more apt

Reply:
Agree. Let me rename it.